Confus/Confus/Audio/PlayerAudioEmitter.cpp

```diff
@@ -14,6 +14,12 @@ namespace Confus
             delete(m_AudioSourceFootstep1);
             delete(m_AudioSourceFootstep2);
             delete(m_AudioSourceFootstep3);
+            delete(m_AudioSourceGrunt1);
+            delete(m_AudioSourceGrunt2);
+            delete(m_AudioSourceSwordSwosh1);
+            delete(m_AudioSourceSwordSwosh2);
+            delete(m_AudioSourceSwordSwosh3);
+            delete(m_AudioSourceSwordSwosh4);
             delete(m_AttachedPlayer);
```

Comment:
I thought this was/would be fixed; deleting memory that isn't allocated by yourself